packages/pnpm/README.md

```diff
@@ -20,6 +20,7 @@ Features:
 * **Strict.** A package can access only dependencies that are specified in its `package.json`.
 * **Deterministic.** Has a lockfile called `pnpm-lock.yaml`.
 * **Works everywhere.** Works on Windows, Linux, and OS X.
+* **Mature.** Used by teams of [all sizes](https://pnpm.js.org/en/users.html) since 2016.
```

Comment:
```suggestion
* **Battle-tested.** Used in production by teams of [all sizes](https://pnpm.js.org/en/users.html) since 2016. To quote the [Rush](https://rushjs.io/) team: “Microsoft uses PNPM in Rush repos with hundreds of projects and hundreds of PRs per day, and we’ve found it to be very fast and reliable.” Need we say more?


```